external_api/es_transfer.py

```diff
@@ -0,0 +1,505 @@
+import requests
```

Comment:
This file belongs inside datastore.elastic_search

Reply:
+1

Reply:
Done